### DISCUSSION TITLE
Blokowanie sklepu net-s

### DISCUSSION THREAD

**Discussion by PolishRoboDog** [FiltersHeroes/KAD#2232] [Pokaż i opowiedz]
### Link bezpośredni 
`https://www.net-s.pl`
### Dodatkowe informacje mogące mieć znaczenie
Dzień dobry,

dlaczego blokujecie w swoich filtrach sklep net-s? Kupuje tam od lat i nie są to żadni oszusci ani sklep skrzak który powstał wczoraj.

Zgłoszenie opublikowane anonimowo przez użytkownika.

**Comment by hawkeye116477** [comment]
#2100
